daemons/dss-events-scribe/app.py

```diff
@@ -28,48 +21,43 @@
 app = domovoi.Domovoi()
 
 
-class ReplicaStatus:
-    prefix = None
-    finished_journaling = False
-    finished_updating = False
+# TODO: Make this configurable
+NUMBER_OF_EVENTS_PER_JOURNAL = 10 if "dev" == os.environ['DSS_DEPLOYMENT_STAGE'] else 1000
 
-    def __init__(self, prefix):
-        self.prefix = prefix
 
+@app.sqs_queue_subscriber("dss-events-scribe-" + os.environ['DSS_DEPLOYMENT_STAGE'],
+                          batch_size=1,
+                          queue_attributes=dict(VisibilityTimeout="600"))
+def handle_sqs_message(event, context):
+    if 1 != len(event['Records']):
+        logger.error(f"Received {len(event['Records'])}, expected 1")
```

Comment:
if the batch_size changes will there be an error? Might want to throw in a comment to that these values can be changed from the lambda console, under lambda triggers

Reply:
This check ensures there is an error if `batch_size != 1`. I'll add a comment.